CMakeLists.txt

```diff
@@ -76,6 +67,14 @@ else()
   set(PAL az_noplatform)
 endif()
 
+add_subdirectory(sdk/core/az_core)
+
+# SDK Clients and tests
+add_subdirectory(sdk/storage/blobs)
```

Comment:
Add a comment of why this reordering is important and shouldn't be changed (PAL needs to come first - why?).

Reply:
This wasn't resolved.

Reply:
Oh sorry I thought you meant on the PR